cmd/bundle/utils/utils.go

```diff
@@ -16,6 +19,30 @@ func configureVariables(cmd *cobra.Command, b *bundle.Bundle, variables []string
 	})
 }
 
+func configureVariablesFromFile(cmd *cobra.Command, b *bundle.Bundle, filePath string) diag.Diagnostics {
+	return bundle.ApplyFunc(cmd.Context(), b, func(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+		f, err := os.ReadFile(filePath)
+		if err != nil {
+			return diag.FromErr(fmt.Errorf("failed to read variables file: %w", err))
+		}
+
+		vars := map[string]any{}
+		err = json.Unmarshal(f, &vars)
+		if err != nil {
+			return diag.FromErr(fmt.Errorf("failed to parse variables file: %w", err))
```

Comment:
If you pass a file with `"foo"` in there, or a list, this will return an error without suggesting what the right format is. Could you look into using `jsonloader` here? It would be nice to return a `diag.Error` with details suggesting what the correct format is, if the contents doesn't match what we expect.